clouddriver-sql/src/main/resources/db/changelog/20210927-accounts.yml

```diff
@@ -0,0 +1,146 @@
+databaseChangeLog:
+- changeSet:
+    id: create-accounts-table
+    author: msicker
+    validCheckSum:
+      # Original changeset checksum
+      - 8:af38865f77cc07c876e92b843be60fd2
```

Comment:
- [ ] Remove changeset checksum; I can nuke the tables in my dev environment with updated version